app/presenters/candidate_interface/application_form_presenter.rb

```diff
@@ -381,7 +381,8 @@ def references
     end
 
     def previous_application_choices_rejected?
```

Comment:
This method name is no longer completely accurate.

Reply:
What do you think it could be? `previous_application_choices_unsuccessful?`? Wdyt?

Reply:
Yeah it's tricky, and I don't want to prolong what is a simple enough bugfix PR, but this opens up the issue of changing the nature of at least 2 components which then don't quite do what their name implies.
Perhaps the most practical thing is to write a tech-debt card for this as there are a few changes here which would greatly increase the scope of the current bug.

So for this PR my only real comment is https://github.com/DFE-Digital/apply-for-teacher-training/pull/7143#discussion_r907321217

Reply:
Done ♻️